app/models/concerns/episode_media.rb

```diff
@@ -195,18 +195,26 @@ def media_duration
     if override?
       external_media_resource&.duration
     else
-      media.inject(0.0) { |s, c| s + c.duration.to_f } + podcast.try(:duration_padding).to_f
+      media_duration_sum + podcast.try(:duration_padding).to_f
```

Comment:
Totally forgot we even had a padding!  Is that normally just null?

Reply:
15 or so? old ones I bet
```
feeder(prod)> Podcast.where("duration_padding is not null").count
=> 15
```